src/emit.h

```diff
@@ -60,8 +63,8 @@ class emit
 
     private:
         emit_id id_;
-        field_id field_ = fd_null;
-        int intensity_ = 3;
+        field_type_id field_ = fd_null;
+        int intensity_ = 1;
```

Comment:
Apparently this requires changes to `emissions.json`, so existing emissions without `intensity` defined should have `intensity` set to `3`